src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -333,9 +335,9 @@ inline std::istream& operator>>(std::istream& is, PerformanceMode& performance_m
 /** @endcond */
 
 /**
- * @brief High-level OpenVINO Performance Hints
- * unlike low-level properties that are individual (per-device), the hints are something that every device accepts
- * and turns into device-specific settings
+ * @brief High-level OpenVINO Performance Hint that configures the runtime for latency-focused or throughput-focused inferencing.
+ * The hint automatically adjusts device and model properties (such as OPTIMAL_NUMBER_OF_INFERENCE_REQUESTS or BATCH_SIZE) 
```

Comment:
can we use references instead of OPTIMAL_NUMBER_OF_INFERENCE_REQUESTS? I mean we want to have automatically generated links once the documentation is built.

Reply:
@ilya-lavrenov @kblaszczak-intel would it be possible to get this merged in 2023.0 documentation? It looks like a great improvement and it will get more out of date if we keep this open for longer. IMO Ilya could just edit the sentences from the last two comments. 
Is it essential to have the snippets in external files or is that something that can be improved later? If it is essential, is there someone from our side who can change that?

Reply:
@ilya-lavrenov @helena-intel 
I believe having code in a document instead of an external snippet should not prevent merging the PR. It may be considered a "better" approach but I think what is even better is actually publishing improvements before articles get deprecated. In-line code works well in our pages, there's no reason not to use it.
When it goes to changes in the src files, I fully agree. I Ilya can adjust the lines how he sees fit, we could merge and publish without further delay.